tests/test_mapdl.py

```diff
@@ -610,3 +617,39 @@ def test_coriolis(mapdl, cleared):
     assert "GYROSCOPIC DAMPING MATRIX WILL BE CALCULATED" in resp
     assert "ROTATING DAMPING MATRIX ACTIVATED" in resp
     assert "PRINT ROTOR MASS SUMMARY ACTIVATED" in resp
+
+
+def test_cdread(mapdl, cleared):
+    random_letters = mapdl.directory.split('/')[0][-3:0]
+
+    mapdl.run(f"parmtest='{random_letters}'")
+    mapdl.cdwrite('all', 'model2', 'db')
+
+    mapdl.clear()
+    mapdl.cdread("db", 'model2', "db")
+
+    assert random_letters == mapdl.parameters['parmtest']
+
+@skip_in_cloud
+def test_cdread_different_location(mapdl, cleared):
```

Comment:
Consider using the `tmpdir` fixture here to create directories.  We could do it from MAPDL, but to me it seems far better to use python directories and pytest fixtures.

```suggestion
def test_cdread_different_location(mapdl, cleared, tmpdir):
```

If you're going to keep using the mapdl directory, don't name the parameters `folder_...`.  "folder" is a GUI term and we should be using either path or directory (hence `os.path.isdir`).